src/app/+my-dspace-page/collection-selector/collection-selector.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { DSpaceObject } from '../../core/shared/dspace-object.model';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'ds-collection-selector',
+  templateUrl: './collection-selector.component.html',
+  styleUrls: ['./collection-selector.component.scss']
+})
+export class CollectionSelectorComponent {
```

Comment:
Please add a TypeDocs description to this class.

Reply:
done